corehq/apps/zapier/views.py

```diff
@@ -69,7 +70,8 @@ def post(self, request, domain, *args, **kwargs):
         else:
             return HttpResponse(status=400)
 
-        return HttpResponse('OK')
+        # respond with the ID so that zapier can use it to unsubscribe
+        return json_response({'id': subscription.id})
```

Comment:
There's also a django built-in now called JsonRepsonse that that behaves pretty much the same. No need to change now, but that reminds me we should probably get rid of this (now superfluous) util.